packages/block-editor/src/components/iframe/index.js

```diff
@@ -190,15 +190,20 @@ function Iframe( { contentRef, children, head, ...props }, ref ) {
 			setBodyClassName( contentDocument );
 			bubbleEvents( contentDocument );
 			setBodyClassName( contentDocument );
-			setIframeDocument( contentDocument );
 			clearerRef( documentElement );
 			clearerRef( body );
 
-			scripts.reduce(
-				( promise, script ) =>
-					promise.then( () => loadScript( contentDocument, script ) ),
-				Promise.resolve()
-			);
+			scripts
+				.reduce(
+					( promise, script ) =>
+						promise.then( () =>
+							loadScript( contentDocument, script )
+						),
+					Promise.resolve()
+				)
+				.finally( () => {
+					setIframeDocument( contentDocument );
```

Comment:
This is needed so `defaultView.Masonry` is available when the block renders. Let's see what impact this has on loading time. Ideally we should render blocks immediately and only delay rendering blocks with script dependencies.